operate/client/playwright.config.ts

```diff
@@ -26,7 +26,11 @@ const getPort = () => {
 };
 
 const getBasePath = () => {
-  return IS_E2E ? '/operate/' : '';
+  if (IS_CI) {
+    return IS_E2E ? '/operate/' : '';
+  }
+
+  return '/operate/';
```

Comment:
What do you think about this? Might make it a bit more readable:

```suggestion
  if (IS_CI && !IS_E2E) {
    return '';
  }

  return '/operate/';
```